frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -113,6 +112,14 @@ class DashboardHeader extends Component {
     addActionToDashboard: PropTypes.func.isRequired,
 
     databases: PropTypes.object,
+
+    dashboardId: PropTypes.number,
+    selectedTabId: PropTypes.number,
+
+    location: PropTypes.shape({
+      query: PropTypes.object,
+      pathname: PropTypes.string,
+    }),
```

Comment:
I saw there were some props appearing within the code that weren't specified here, so added them